src/visip/action/std.py

```diff
@@ -132,9 +132,9 @@ def format(format_str: str, *args: Any) -> str:
     return format_str.format(*args)
 
 @decorators.action_def
-def file_from_template(template: dtype.Constant[FileIn],
-                       parameters: dtype.Dict[str, str],
-                       delimiters: dtype.Constant[str]="<>") -> FileIn:
+def file_from_template(template: dtype.Const(FileIn),
+                       parameters: Dict[str, str],
```

Comment:
Keep 'dtype.Dict' here and there.
Idea is to use visip types in the visip code and typing types in imported Python functions through the 'action_def' decorator.